tests/test_incremental_pca.py

```diff
@@ -0,0 +1,443 @@
+"""Tests for Incremental PCA."""
+import numpy as np
+import pytest
+from dask import array as da
+
+from sklearn.utils._testing import assert_almost_equal
+from sklearn.utils._testing import assert_array_almost_equal
+from sklearn.utils._testing import assert_allclose_dense_sparse
```

Comment:
Seems to be causing an import error. Can these be avoided?

Reply:
Just noticed that older scikit-learn does not have `utils._testing` but instead `utils.testing`.
Added a patch by `try: except ImportError`.